app.yaml

```diff
@@ -219,3 +219,48 @@ skip_files:
 - third_party/static/rangy-1.3.0
 - third_party/static/textAngular-1.4.5
 - third_party/static/widget
+- third_party/static/MathJax-2.6.0/docs
```

Comment:
Also, are these supposed to change in this PR?

Reply:
I am not sure how these entries got added but I checked out a new copy from upstream! 👍